unittests/tools/test_qualys_parser.py

```diff
@@ -13,6 +13,43 @@ def test_parse_file_with_no_vuln_has_no_findings(self):
         findings = parser.get_findings(testfile, Test())
         self.assertEqual(0, len(findings))
 
+    def test_parse_file_with_cvss_values_and_scores(self):
+        testfile = open(
+            get_unit_tests_path() + "/scans/qualys/Qualys_Sample_Report.xml"
+        )
+        parser = QualysParser()
+        findings = parser.get_findings(testfile, Test())
+        for finding in findings:
+            for finding in findings:
```

Comment:
This double loop seems unnecessary?

```suggestion
        for finding in findings:
```